packages/protocol/contracts/ACE/validators/swap/Swap.sol

```diff
@@ -172,6 +172,8 @@ contract Swap {
                     if iszero(result) { mstore(0x00, 400) revert(0x00, 0x20) }
                     b := add(b, 0x40) // increase B pointer by 2 words
                 }
+                validatePairing(0x84)
```

Comment:
@zac-williamson @thomas-waite wanted this highlighted can you review as well.

Reply:
As discussed, merging now and have added a task to discuss